README.md

```diff
@@ -92,137 +54,110 @@ After you're finished please remove all the comments and instructions!
   </ul>
 </details>
 
-<!-- Features -->
-
 ### Key Features <a name="key-features"></a>
 
 > Describe between 1-3 key features of the application.
 
```

Comment:
- [ ] Kindly remove this section description because it is just an instruction for you to make your readme file once you make it then these instructions are no more useful 🤔